IceCubesApp/Resources/Localization/pt-BR.lproj/Localizable.strings

```diff
@@ -291,6 +291,7 @@
 
 // MARK: Package: Status
 "status.action.translate" = "Traduzir";
+"status.action.translate-from-%@" = "Translate from %@";
```

Comment:
shouldn't this be in portuguese?
```suggestion
"status.action.translate-from-%@" = "Traduzir de %@";
```

Reply:
Yeah, what I added (except for the Korean file) are just English placeholders.
I believe the translation works have been done this way so far (English placeholder added → translated by the speakers).

Will it be okay if I commit your suggestion for this Portuguese one? Please let me know.

Reply:
Sorry, missed this and just saw you already merged, but yeah, perfectly fine to commit this suggestion 🙂